packages/apputils-extension/style/index.css

```diff
@@ -4,5 +4,6 @@
 | Distributed under the terms of the Modified BSD License.
 |----------------------------------------------------------------------------*/
 
-@import './redirect.css';
-@import './splash.css';
+@import './base.css';
+
+@import url('~@jupyterlab/apputils/style/index.css');
```

Comment:
In general, I think the dependencies should be imported first, and *then* the base.css, so that the extension has a chance to override things in the dependency css if needed.

Reply:
Part of the follow up task is to formalise this during the rc phase

Reply:
And automate it

Reply:
sounds good.